contributing_to_docs/create_or_edit_content.adoc

```diff
@@ -171,7 +171,7 @@ that specifies the minimum version that the change applies to. The docs team
 maintains these branches for all active and future distros and your PR will be
 applied to one or more of these branches.
 *** PR applies to all versions after a specific version (e.g. 4.8): 4.8+
-*** PR applies to the in-development version (e.g. 4.12) and future versions: 4.12+
+*** PR applies to the in-development version (e.g. 4.13) and future versions: 4.13+
```

Comment:
The contrib docs should only be updated in main, so this change should be reverted.